manifests/params.pp

```diff
@@ -32,6 +32,8 @@
 
   $crl_file = '/var/lib/candlepin/candlepin-crl.crl'
 
+  $address = '127.0.0.1'
```

Comment:
The default is to listen on all IPs and I'd prefer to keep it that way so we don't have to do a major version bump. I don't know if that's equivalent to ```0.0.0.0``` or if it also listens on IPv6. Then ```::``` might be the same. If that doesn't work then it should allow ```undef``` to maintain current behavior.

Reply:
@ekohl the goal of the change is to correct this bz: https://bugzilla.redhat.com/show_bug.cgi?id=1501499 so what would be a better way of changing this be?

Reply:
You can override it in puppet-katello. You will also need to ensure that a certificate is generated with the name localhost. I think others do that via the certs module answer file so it might also need a fix in katello-installer.

The best way of looking at it is that puppet-candlepin provides a very generic module with sane defaults for a candlepin user. In puppet-katello we ensure it's suited for the deployment we want in a katello setup.

Reply:
Also note, in prior versions we connected Katello to Candlepin over localhost but switched to using hostname. If we switch back to localhost, then as @ekohl points out, we have to do some certificate regeneration to include a new Subject Alt-Name for localhost. This is similar to when qpid was locked down to localhost only. We also want to ensure hostname continues to work in the cases where users may want to split out CP eventually.